bin/build.cmd

```diff
@@ -0,0 +1 @@
+@go install
```

Comment:
Why does this need an `@` sign while `bin/setup.cmd` doesn't for `go get` and `gometalinter.v1`?

Reply:
A preceding `@` omits the command itself being printed before it is run. I thought seeing what `setup` runs is useful, but the user doesn't want to see the build command all the time, since it is not that relevant.